Bio/PDB/FragmentMapper.py

```diff
@@ -88,12 +87,9 @@ def _read_fragments(size, length, dir="."):
 
 
 class Fragment(object):
-    """
-    Represent a polypeptide C-alpha fragment.
-    """
+    """Represent a polypeptide C-alpha fragment."""
     def __init__(self, length, fid):
-        """
-        @param length: length of the fragment
+        """@param length: length of the fragment
```

Comment:
About about using ``Initialise`` as a (better than nothing) one line summary, and leaving the parameters in the main body of the docstring as before?